### PR TITLE
nRF21540 runtime MODE pin switching

### PR DESCRIPTION
**Pull request by ankuns**
This PR brings ability to control gain of nRF21540 in run-time by switching `MODE` pin.
The feature is implemented in mpsl, changes in the sdk-nrf allow this feature to be enabled by an user.

mpsl will go with https://github.com/nrfconnect/sdk-nrf/pull/7578, so merge this PR after that

- [x] Merge after https://github.com/nrfconnect/sdk-nrf/pull/7578
- [x] Remove commit "manifest: Update nrfxlib revision" after #7578 is merged

### CODE REVIEW

**Review by ahasztag — APPROVED** [review]

**Review by piotrslawecki — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

File: doc/nrf/ug_radio_fem.rst [hunk 1]
```diff
@@ -312,7 +325,7 @@ To use the Simple GPIO implementation of FEM with SKY66112-11, complete the foll
 Optional properties
 -------------------
 
-The following properties are optional and you can add them to the devicetree node if needed.
+The following properties are optional and   you can add them to the devicetree node if needed.
```
Comment (jciupis) [inline doc/nrf/ug_radio_fem.rst:328]: @FrancescoSer is this change intentional?

**Comment by ankuns** [comment]
just rebased

**Comment by ankuns** [comment]
rebased
